README.md

```diff
@@ -89,6 +89,7 @@ Collection of awesome Python resources for testing and generating test data.
 ## Mutation Testing
 
 - [Cosmic Ray](https://github.com/sixty-north/cosmic-ray) - makes small changes to your source code, running your test suite for each one.
+- [Mutatest](https://mutatest.readthedocs.io/en/latest) - Python mutation testing.
```

Comment:
```suggestion
- [Mutatest](https://github.com/EvanKepner/mutatest) - Python mutation testing.
```

Reply:
Okay, I agree to be the (GitHub) repository link as standard